packages/table/package.json

```diff
@@ -19,7 +19,7 @@
     "watch": "onchange 'src/**' 'preview/*.ts*' -- npm-run-all build:gulp build:preview"
   },
   "dependencies": {
-    "@blueprintjs/core": "^1.2.0",
+    "@blueprintjs/core": "^1.4.0",
```

Comment:
Please revert this line; it is not accurate and the current version range already allows 1.4